recipe/meta.yaml

```diff
@@ -19,27 +19,36 @@ requirements:
     - python >=3.8
     - pip
     - pytest-runner >=5.2
+    - setuptools
   run:
     - python >=3.8
-    - dask >=2021.4.1
+    - dask >=2021.4.1,<=2023.5.0
     - fsspec >=2022.8.0
     - imagecodecs >=2020.5.30
-    - lxml >=4.6
-    - numpy >=1.16
-    - ome-types >=0.3.3
+    - lxml >=4.6,<5
+    - numpy >=1.16,<=1.24.0
+    # ome-types 0.3.4 is one higher than the aicsimageio pip package for 4.12.0, however
+    # 0.3.3 has broken since the Pydantic v2 release
+    - ome-types >=0.3.4,<0.4.0
```

Comment:
Ah, you didn't mention this mismatch when you asked for v0.3.4

Conda actually lets you make revisional back-releases, so I can release v0.3.3 with a different pinning if that's what the issue was.  Has this version actually been well tested with ome types 0.3.4?

Reply:
I possibly didn't explain myself well. [The pip dependencies for `aicsimageio` 4.12 are](https://github.com/AllenCellModeling/aicsimageio/blob/v4.12.0/setup.py#L108) `"ome-types>=0.3.3,<0.4.0"`. So I increased the minimum allowed version by one. 0.3.4 is still within the bounds of the official release, only with tighter bounds.

I had to do this because `ome-types` 0.3.3 allows Pydantic 2.X which breaks everything.

Reply:
ah, ok, should be fine. 

> I had to do this because ome-types 0.3.3 allows Pydantic 2.X which breaks everything.

yeah, understood.  just wanted to mention that, unlike pip, conda does let you go back and repin a historical version.  So I can effectively go back and disallow pydantic 2 on v0.3.3

Reply:
cool thanks

Reply:
I guess the thing is that we have to rewrite history somewhere. It could be in `ome-types` or it could be in `aicsimageio`, and I'm not sure if it exactly matters where.

Reply:
It would only be in `ome-types-feedstock` (at least as far as what I'm picturing)